### ISSUE TITLE
Error if trace = TRUE in ashci

### ISSUE THREAD

**Issue by frederikziebell** [stephens999/ashr#133]
This
```
beta <-  c(rep(0,20),rnorm(20))
sebetahat <-  abs(rnorm(40,0,1))
betahat <-  rnorm(40,beta,sebetahat)
beta.ash <-  ash(betahat, sebetahat)
res <- ashci(beta.ash,betaindex = 1:10, trace = TRUE)
```
gives an error
```
Error in assign(".type", type, envir = baseenv()) : 
cannot add binding of '.type' to the base environment
```
which does not appear, if `trace = FALSE` is set

**Comment by pcarbo** [comment]
@frederikziebell I could not reproduce your error. Which version of the ashr package are you using? Maybe try with `library(ashr)` rather than, say, loading the package with `devtools::load_all()`?

**Comment by stephens999** [comment]
this appears to be due to a breaking change in R v 4.1.0 which 
had the following change:
> The base environment and its namespace are now locked (so one can no longer add bindings to these or remove from these).

This lead to breaking of the ashr `tic` and `toc` functions.
I've replaced these functions with the latest commit

**Comment by stephens999** [comment]
fixed with 9c5a495
